index.html

```diff
@@ -0,0 +1,154 @@
+                <li class="inner-list"><img src="images/twitter.png" alt=""></li>
+                <li class="inner-list"><a href="#">English</a></li>
+                <li class="inner-list"><a href="#">My page</a></li>
+                <li class="inner-list"><a href="#">Logout</a></li>
+            </nav>
+        </div>
+            <header>
+                <div class="main-containers">
+                    <div class="header-container">
+                      <nav class="header">
+                        <div class="my-logo">
+                            <a href="#"><img class="logo" src="images/Ifeanyi_Logo_1B-01-removebg-preview.png" alt=""></a>
+                        </div>
+                        <div class="hambuger">
+                          <div class="bar"></div>
+                          <div class="bar"></div>
+                          <div class="bar"></div>
+                        </div>
+                        <ul class="nav-list">
+                          <li class="nav-item about"><a href="about.html">About</a></li>
+                          <li class="nav-item"><a href="#section1">Program</a></li>
+                          <li class="nav-item"><a href="#section2">join</a></li>
+                          <li class="nav-item"><a href="#section3">Sponsor</a></li>
+                          <li class="nav-item"><a href="#">News</a></li>
+                          <li class="nav-item"><a class="campaign" href="#">Campaign</a></li>
+                        </ul>
+                      </nav>
+                    </div>
+                </div>
+            </header>
+    </section>
+    <main>
+        <section class="title-section">
+            <div class="main-container">
+                <div class="page-title">
+                    <span class="greetings">"Hello! friends"</span>
+                    <h1 class="learning">Ifeanyi305 Virtual Learning in Schools: Pros and Cons.</h1>
+                    <div class="description">
+                        This is an online platform for education whereby students can study remotely 
+                        with the help of either thier mobile phones,computer or laptop 
+                        using the internet. join us.
+                    </div>
+                    <div class="time">
+                    <h1>2022.10.14(Thu) ~ 17(Fri)</h1>
+                    <p>National museum of korea. Art Centre Nabi and more</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <section class="main-program-section" id="section1">
+            <div class="main-container">
+                <span class="main-program">Main program</span>
+                <hr class="hr-line">
+                <div class="program-img">
+                    <div class="lecture">
+                        <img src="images/program_icon_01.png" alt="">
+                        <p class="sub-heading">lecture</p>
+                        <span class="text">
```

Comment:
- [ ] Kindly adjust your styling so that your design looks more close to the given design
- I tried changing the following values and it worked, maybe you could try it
![image](https://user-images.githubusercontent.com/81932478/196739183-3849fef2-6dae-4cdd-b606-013628ae93c5.png)
This was the outcome
![image](https://user-images.githubusercontent.com/81932478/196739472-a99ce64c-7a29-441e-8df3-f48e6974700f.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/196739608-7c25efb2-e529-40e3-b56a-26e8f4f4fe22.png)

Your current style setting
![image](https://user-images.githubusercontent.com/81932478/196739788-9de1c370-ebaf-4a06-bc3c-86bb6f7eeed1.png)